RecoMET/METFilters/python/BadPFMuonFilter_Dz_cfi.py

```diff
@@ -0,0 +1,19 @@
+import FWCore.ParameterSet.Config as cms
+
+BadPFMuonFilterDz = cms.EDFilter(
```

Comment:
the file name does not match the instance name inside it; please rename to ```badPFMuonFilterDz_cfi.py```
Also, now that the fillDescriptions is available, please clone from it instead of redefining a new config
```python
import  RecoMET.METFilters.BadPFMuonFilter_cfi as _mod
badPFMuonFilterDz = _mod.BadPFMuonFilter.clone(
  filterType = "BadPFMuonDz",
)
```
and replace other parameters, different from the original module

Reply:
renamed to BadPFMuonDzFilter_cfi.py to be consistent with other names